nodeup/pkg/model/context.go

```diff
@@ -19,10 +19,42 @@ package model
 import (
 	"k8s.io/kops/nodeup/pkg/distros"
 	"k8s.io/kops/pkg/apis/kops"
+	"k8s.io/kops/upup/pkg/fi"
 )
 
 type NodeupModelContext struct {
```

Comment:
I think I lost the line you were referring to @hingstarne , probably in a rebase?